.github/workflows/build_playground_backend.yml

```diff
@@ -81,21 +81,30 @@ jobs:
       - name: Preapre Build Go
```

Comment:
@andreykus (cc: @daria-malkova) Please correct spelling of 'Prepare' in a future PR.